lib/connectors_app/config.rb

```diff
@@ -3,14 +3,42 @@
 # or more contributor license agreements. Licensed under the Elastic License;
 # you may not use this file except in compliance with the Elastic License.
 #
-require 'yaml'
+
+require 'config'
+
+# We look for places in this order:
+# - CONNECTORS_CONFIG environment variable
+# - here: /../../config/connectors.yml
+CONFIG_FILE = ENV['CONNECTORS_CONFIG'] || File.join(__dir__, '../..', 'config', 'connectors.yml')
+
+puts "Parsing #{CONFIG_FILE} configuration file."
+
+Config.setup do |config|
```

Comment:
Lets make this `::Config`, so that it's really explicit when we use `::Config` vs `ConnectorsApp::Config` in this file.